packer/http/ubuntu-20.04/preseed.cfg

```diff
@@ -0,0 +1,40 @@
+choose-mirror-bin mirror/http/proxy string
```

Comment:
This is an exact copy of the 18.04 preseed file.

Reply:
🤷